### PR TITLE
OAEWDGT-68 - Add a :focus to each :hover

### PR DESCRIPTION
**Pull request by christianvuerings**
Jira: https://jira.sakaiproject.org/browse/OAEWDGT-68

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -706,7 +707,7 @@ textarea {
 /******************/
 /* DROPDOWN LISTS */
 /******************/
-.wl-dropdown-menu:hover, .wl-dropdown-menu.opened {
+.wl-dropdown-menu:hover, .wl-dropdown-menu:focus, .wl-dropdown-menu.opened {
```
Comment (croby) [inline app/assets/stylesheets/application.css.scss]: Let's convert this to use the `&` notation like above, so this becomes

``` sass
.wl-dropdown-menu {
  &:hover, &:focus, &.opened {
  }  }
```

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -775,7 +776,7 @@ textarea {
       width: 110px;
       border-right: 2px solid #fff;
       float: left; }
-    li.navigation_item.selected, li.navigation_item:hover {
+    li.navigation_item.selected, li.navigation_item:hover, li.navigation_item:focus {
```
Comment (croby) [inline app/assets/stylesheets/application.css.scss]: Same as above

File: app/assets/stylesheets/application.css.scss (hunk 1 above)
Comment (christianvuerings) [inline app/assets/stylesheets/application.css.scss]: Ok

File: app/assets/stylesheets/application.css.scss (hunk 2 above)
Comment (christianvuerings) [inline app/assets/stylesheets/application.css.scss]: Also did some more cascading in the updated commit

**Pull request merged**